packages/knip/src/plugins/bumpp/index.ts

```diff
@@ -0,0 +1,19 @@
+import type { IsPluginEnabled, Plugin } from '../../types/config.js';
+import { hasDependency } from '../../util/plugin.js';
+
+// https://github.com/antfu-collective/bumpp#bumpp
+
+const title = 'bumpp';
+
+const enablers = ['bumpp'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => hasDependency(dependencies, enablers);
+
+const entry = ['bump.config.{mjs,ts,js,cjs,mts,cts}'];
```

Comment:
The docs only mention a `bump.config.ts` but I tested it with the other extensions and they also work, which makes sense since it's using the [`c12`](https://github.com/unjs/c12) config loader

Reply:
Thanks for looking that up. You could also consider using the `toC12config` helper (leaving that up to you).